x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/use_conversation/index.tsx

```diff
@@ -129,8 +129,8 @@ export const useConversation = (): UseConversation => {
   const setApiConfig = useCallback(
     async ({ conversation, apiConfig }: SetApiConfigProps) => {
       if (conversation.id === '') {
-        // only developer should ever see this error
-        throw new Error('Conversation ID is required to set API config');
+        // Conversation ID is required to set API config, return empty conversation
```

Comment:
intended the developer to see this error and not commit silly code, but of course this error instead broke the flow because the developer didn't test. the flow... needs cypress tests